sdk/src/types/block/output/feature/metadata.rs

```diff
@@ -31,6 +31,14 @@ impl TryFrom<Vec<u8>> for MetadataFeature {
     }
 }
 
+impl FromStr for MetadataFeature {
+    type Err = Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        Self::new(prefix_hex::decode::<Vec<u8>>(s).map_err(Error::Hex)?)
```

Comment:
Could change `new` to take `impl Into<Box<[u8]>>` while you're in here.

Reply:
I don't know about that

Reply:
Doing the box breaks some nice usecase like `MetadataFeature::new("Hello, World!")`

Reply:
Can't, needs to be fallible because of the size